gossip/gossip.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package gossip
```

Comment:
I wonder if we should either put every gossip related file here, or rename this package. Additionaly we can move this under plugin/evm as a subpackage of `gossip`. like plugin/evm/gossip/bloomfilter(or pullgossip)

Reply:
Yeah I didn't do this originally since all the other gossip types lived in weird parts of this repo. I think it makes sense to self contain this though to make porting easier. I'll see if I can move everything here.

Reply:
Refactored everything to be self-contained in gossip